core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -88,6 +89,7 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
   selectedTopicNames: string[] = [];
   languageChoices: LanguageChoice[] = [];
   topics: TopicChoice[] = [];
+  topicsFetched: string = 'false';
```

Comment:
I've renamed the variable, but I can't use a boolean value, as in the component.html it is found to have a value of TRUE (because it exists), even if the value defined in componenet.ts FALSE. Hope this makes sense, feel free to get in touch to discuss if it doesn't :)